### ISSUE TITLE
Connection to runner with Pid 19604 lost. The runner probably quit unexpectedly. Inspect logs for potential reasons. Error : read tcp 127.0.0.1:50374->127.0.0.1:50377: use of closed network connection

### ISSUE THREAD

**Issue by kokavolchkov** [getgauge/gauge-java#381]
### Expected behavior
The example.spec is run successfully 

### Actual behavior
The "Connection to runner with Pid 19604 lost. The runner probably quit unexpectedly. Inspect logs for potential reasons. Error : read tcp 127.0.0.1:50374->127.0.0.1:50377: use of closed network connection" error appears when I run the example.spec 

### Steps to reproduce
1. Install the Intellij Idea on the PC (Windows 10)
2. Install the Gauge 
3. Install the Gauge extension 
4. Create the Gauge project
5. Select the example.spec and run it
6. Pay attention to the Run result:

[Gauge]
Connection to runner with Pid 44568 lost. The runner probably quit unexpectedly. Inspect logs for potential reasons. Error : read tcp 127.0.0.1:59256->127.0.0.1:59259: use of closed network connection

### Gauge version
1.0.6
Commit Hash: 2bc49db

Plugins
-------
dotnet (0.1.2)
html-report (4.0.8)
java (0.7.3)
screenshot (0.0.1)

### IntelliJ IDEA
IntelliJ IDEA 2019.3.2 (Community Edition)
Build #IC-193.6015.39, built on January 21, 2020
Runtime version: 11.0.5+10-b520.30 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 1963M
Cores: 8
Registry: 
Non-Bundled Plugins: com.thoughtworks.gauge

### gauge --version --machine-readable
{
    "version": "1.0.6",
    "commitHash": "2bc49db",
    "plugins": [
        {
            "name": "dotnet",
            "version": "0.1.2"
        },
        {
            "name": "html-report",
            "version": "4.0.8"
        },
        {
            "name": "java",
            "version": "0.7.3"
        },
        {
            "name": "screenshot",
            "version": "0.0.1"
        }
    ],
    "telemetry": "This installation of Gauge collects usage data in order to help us improve your experience.\nThe data is anonymous and doesn't include command-line arguments.\nTo turn this message off opt in or out by running 'gauge telemetry on' or 'gauge telemetry off'.\n\nRead more about Gauge telemetry at https://gauge.org/telemetry\n"
}

### Log
24-02-2020 08:51:20.341 [Gauge] [DEBUG] Plugin java is already installed.
24-02-2020 08:51:20.351 [Gauge] [DEBUG] Plugin html-report is already installed.
24-02-2020 08:51:20.490 [java] [DEBUG] classpath set to C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.base;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.compiler;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.datatransfer;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.desktop;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.instrument;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.logging;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.management;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.management.rmi;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.naming;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.net.http;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.prefs;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.rmi;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.scripting;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.se;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.security.jgss;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.security.sasl;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.smartcardio;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.sql;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.sql.rowset;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.transaction.xa;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.xml;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.xml.crypto;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.base;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.controls;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.fxml;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.graphics;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.media;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.swing;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.web;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.accessibility;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.aot;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.attach;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.charsets;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.compiler;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.crypto.cryptoki;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.crypto.ec;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.dynalink;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.hotspot.agent;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.httpserver;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.ed;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.jvmstat;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.le;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.vm.ci;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.vm.compiler;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.vm.compiler.management;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.jdi;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.jdwp.agent;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.jfr;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.jsobject;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.localedata;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.management;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.management.agent;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.management.jfr;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.naming.dns;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.naming.rmi;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.net;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.pack;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.scripting.nashorn;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.scripting.nashorn.shell;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.sctp;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.security.auth;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.security.jgss;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.unsupported;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.xml.dom;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.zipfs;C:\Users\user\source\repos\FS Automation\Java\out\test\Java;C:\Users\user\source\repos\FS Automation\Java\out\production\Java;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\gson-2.8.5.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\jsr305-3.0.2.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\json-20190722.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\commons-io-2.6.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\guava-28.1-jre.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-api-1.25.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\commons-lang-2.6.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\gauge-java-0.7.3.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-core-1.25.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-stub-1.24.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\checker-qual-2.8.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\reflections-0.9.11.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\annotations-4.1.1.4.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\assertj-core-3.14.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\failureaccess-1.0.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-context-1.25.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\javassist-3.21.0-GA.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\perfmark-api-0.19.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-protobuf-1.24.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\protobuf-java-3.10.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\javaparser-core-2.5.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\opencensus-api-0.21.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\j2objc-annotations-1.3.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-netty-shaded-1.25.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-protobuf-lite-1.24.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\javax.annotation-api-1.3.2.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\error_prone_annotations-2.3.2.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\animal-sniffer-annotations-1.18.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\proto-google-common-protos-1.12.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\opencensus-contrib-grpc-metrics-0.21.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar;C:\Users\user\source\repos\FS Automation\Java\selenium-java-3.141.59\client-combined-3.141.59.jar;C:\Users\user\source\repos\FS Automation\Java\selenium-java-3.141.59\client-combined-3.141.59-sources.jar
24-02-2020 08:51:20.498 [Gauge] [DEBUG] Checking updates...
24-02-2020 08:51:20.500 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/dotnet?l=java&p=dotnet,html-report,java,screenshot&o=windows&a=amd64
24-02-2020 08:51:21.316 [java] [DEBUG] Connecting to port: 59256
24-02-2020 08:51:21.324 [java] [DEBUG] Connected to 127.0.0.1:59256
24-02-2020 08:51:21.422 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/html-report?l=java&p=dotnet,html-report,java,screenshot&o=windows&a=amd64
24-02-2020 08:51:21.644 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/java?l=java&p=dotnet,html-report,java,screenshot&o=windows&a=amd64
24-02-2020 08:51:21.870 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/screenshot?l=java&p=dotnet,html-report,java,screenshot&o=windows&a=amd64
24-02-2020 08:51:23.532 [java] [DEBUG] Scanning packages for steps
24-02-2020 08:51:23.548 [java] [CRITICAL] Error in thread 1
Can't resolve member named navigateTo for class EndToEndTest
org.reflections.ReflectionsException: Can't resolve member named navigateTo for class EndToEndTest
at org.reflections.util.Utils.getMemberFromDescriptor(Utils.java:94)
at org.reflections.util.Utils.getMethodsFromDescriptors(Utils.java:101)
at org.reflections.Reflections.getMethodsAnnotatedWith(Reflections.java:482)
at com.thoughtworks.gauge.scan.StepsScanner.scan(StepsScanner.java:41)
at com.thoughtworks.gauge.scan.ClasspathScanner.scan(ClasspathScanner.java:48)
at com.thoughtworks.gauge.GaugeRuntime.connectSynchronously(GaugeRuntime.java:108)
at com.thoughtworks.gauge.GaugeRuntime.startGaugeServer(GaugeRuntime.java:76)
at com.thoughtworks.gauge.GaugeRuntime.main(GaugeRuntime.java:53)
24-02-2020 08:51:23.604 [Gauge] [CRITICAL] Connection to runner with Pid 44568 lost. The runner probably quit unexpectedly. Inspect logs for potential reasons. Error : read tcp 127.0.0.1:59256->127.0.0.1:59259: use of closed network connection

**Comment by kokavolchkov** [comment]
### Log 

24-02-2020 08:51:20.341 [Gauge] [DEBUG] Plugin java is already installed.
24-02-2020 08:51:20.351 [Gauge] [DEBUG] Plugin html-report is already installed.
24-02-2020 08:51:20.490 [java] [DEBUG] classpath set to C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.base;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.compiler;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.datatransfer;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.desktop;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.instrument;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.logging;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.management;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.management.rmi;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.naming;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.net.http;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.prefs;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.rmi;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.scripting;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.se;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.security.jgss;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.security.sasl;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.smartcardio;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.sql;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.sql.rowset;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.transaction.xa;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.xml;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\java.xml.crypto;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.base;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.controls;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.fxml;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.graphics;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.media;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.swing;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\javafx.web;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.accessibility;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.aot;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.attach;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.charsets;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.compiler;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.crypto.cryptoki;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.crypto.ec;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.dynalink;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.hotspot.agent;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.httpserver;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.ed;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.jvmstat;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.le;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.vm.ci;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.vm.compiler;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.internal.vm.compiler.management;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.jdi;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.jdwp.agent;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.jfr;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.jsobject;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.localedata;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.management;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.management.agent;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.management.jfr;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.naming.dns;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.naming.rmi;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.net;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.pack;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.scripting.nashorn;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.scripting.nashorn.shell;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.sctp;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.security.auth;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.security.jgss;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.unsupported;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.xml.dom;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.2\jbr!\jdk.zipfs;C:\Users\user\source\repos\FS Automation\Java\out\test\Java;C:\Users\user\source\repos\FS Automation\Java\out\production\Java;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\gson-2.8.5.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\jsr305-3.0.2.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\json-20190722.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\commons-io-2.6.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\guava-28.1-jre.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-api-1.25.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\commons-lang-2.6.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\gauge-java-0.7.3.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-core-1.25.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-stub-1.24.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\checker-qual-2.8.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\reflections-0.9.11.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\annotations-4.1.1.4.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\assertj-core-3.14.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\failureaccess-1.0.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-context-1.25.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\javassist-3.21.0-GA.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\perfmark-api-0.19.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-protobuf-1.24.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\protobuf-java-3.10.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\javaparser-core-2.5.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\opencensus-api-0.21.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\j2objc-annotations-1.3.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-netty-shaded-1.25.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\grpc-protobuf-lite-1.24.1.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\javax.annotation-api-1.3.2.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\error_prone_annotations-2.3.2.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\animal-sniffer-annotations-1.18.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\proto-google-common-protos-1.12.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\opencensus-contrib-grpc-metrics-0.21.0.jar;C:\Users\user\AppData\Roaming\gauge\plugins\java\0.7.3\libs\listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar;C:\Users\user\source\repos\FS Automation\Java\selenium-java-3.141.59\client-combined-3.141.59.jar;C:\Users\user\source\repos\FS Automation\Java\selenium-java-3.141.59\client-combined-3.141.59-sources.jar
24-02-2020 08:51:20.498 [Gauge] [DEBUG] Checking updates...
24-02-2020 08:51:20.500 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/dotnet?l=java&p=dotnet,html-report,java,screenshot&o=windows&a=amd64
24-02-2020 08:51:21.316 [java] [DEBUG] Connecting to port: 59256
24-02-2020 08:51:21.324 [java] [DEBUG] Connected to 127.0.0.1:59256
24-02-2020 08:51:21.422 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/html-report?l=java&p=dotnet,html-report,java,screenshot&o=windows&a=amd64
24-02-2020 08:51:21.644 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/java?l=java&p=dotnet,html-report,java,screenshot&o=windows&a=amd64
24-02-2020 08:51:21.870 [Gauge] [DEBUG] Downloading https://downloads.gauge.org/plugin/screenshot?l=java&p=dotnet,html-report,java,screenshot&o=windows&a=amd64
24-02-2020 08:51:23.532 [java] [DEBUG] Scanning packages for steps
24-02-2020 08:51:23.548 [java] [CRITICAL] Error in thread 1
Can't resolve member named navigateTo for class EndToEndTest
org.reflections.ReflectionsException: Can't resolve member named navigateTo for class EndToEndTest
	at org.reflections.util.Utils.getMemberFromDescriptor(Utils.java:94)
	at org.reflections.util.Utils.getMethodsFromDescriptors(Utils.java:101)
	at org.reflections.Reflections.getMethodsAnnotatedWith(Reflections.java:482)
	at com.thoughtworks.gauge.scan.StepsScanner.scan(StepsScanner.java:41)
	at com.thoughtworks.gauge.scan.ClasspathScanner.scan(ClasspathScanner.java:48)
	at com.thoughtworks.gauge.GaugeRuntime.connectSynchronously(GaugeRuntime.java:108)
	at com.thoughtworks.gauge.GaugeRuntime.startGaugeServer(GaugeRuntime.java:76)
	at com.thoughtworks.gauge.GaugeRuntime.main(GaugeRuntime.java:53)
24-02-2020 08:51:23.604 [Gauge] [CRITICAL] Connection to runner with Pid 44568 lost. The runner probably quit unexpectedly. Inspect logs for potential reasons. Error : read tcp 127.0.0.1:59256->127.0.0.1:59259: use of closed network connection

**Comment by BugDiver** [comment]
@kokavolchkov Can you please upgrade `gauge` and `gauge-java` to the latest versions and check if that fixes the issue? Also if you are using maven/Gradle please upgrade `gauge-java` dependency version as well.

**Comment by kokavolchkov** [comment]
@BugDiver I have updated the gauge version:

```
Gauge version: 1.0.7
Commit Hash: ed7b4fd6
Plugins
-------
dotnet (0.1.3)
html-report (4.0.9)
java (0.7.4)
screenshot (0.0.1)
```
**And this error message is gone:** "Connection to runner with Pid 19604 lost. The runner probably quit unexpectedly. Inspect logs for potential reasons. Error : read tcp 127.0.0.1:50374->127.0.0.1:50377: use of closed network connection"

**But.** Update leads to this error: https://github.com/getgauge/gauge/issues/1591

**So, to sum up:** In my case the "Connection to runner with Pid 19604 lost..." error was fixed by an update from Gauge 1.0.6 to 1.0.7
